chia/seeder/peer_record.py

```diff
@@ -20,6 +20,7 @@ class PeerRecord(Streamable):
     best_timestamp: uint64
     version: str
     handshake_time: uint64
+    tls_version: str
```

Comment:
You don't think it would be nicer to make this `Optional[str]`, and instead of populating it with "unknown", just set it to `None` of we don't know.